python-sdk/nuscenes/eval/predict/splits.py

```diff
@@ -0,0 +1,37 @@
+# nuScenes dev-kit.
+# Code written by Freddy Boulton.
+
+import json
+import os
+from itertools import chain
+from typing import List
+
+from nuscenes.utils.splits import create_splits_scenes
+
+
+def get_prediction_challenge_split(split: str) -> List[str]:
+    """
+    Gets a list of {instance_token}_{sample_token} strings for each split
+    :param split: One of 'mini_train', 'mini_val', 'train', 'val'.
+    """
+    if split not in {'mini_train', 'mini_val', 'train', 'train_val', 'val'}:
+        raise ValueError("split must be one of (mini_train, mini_val, train, train_val, val)")
+    
+    if split == 'train_val':
+        split_name = 'train'
+    else:
+        split_name = split
+
+    num_in_val = 200
```

Comment:
Make this a parameter

Reply:
I expect this to be a constant and not passed in by the user. I can move it to the module scope and make the name all caps to make that clearer. What are your thoughts?